libr/bin/format/mz/mz.c

```diff
@@ -78,6 +75,10 @@ struct r_bin_mz_segment_t * r_bin_mz_get_segments(
 		btree_add (&tree, (void *)&stack_segment, cmp_segs);
 	}
 
+	if (num_relocs * sizeof(*segments) ==  0) {
```

Comment:
I was trying to maintain the same semantic as one line after that notepad wrote. 
`segments = calloc (num_relocs, sizeof(*segments));`

I think that in this case sizeof(*segments) is sizeof(ut16), sizeof(segments) will be over a pointer instead. But I could be wrong.